app/src/main/java/com/mashup/lastgarden/ui/scent/ScentViewModel.kt

```diff
@@ -15,30 +18,52 @@ class ScentViewModel @Inject constructor(
     private val storyRepository: StoryRepository
 ) : ViewModel() {
 
-    private var _sortOrder = MutableLiveData(Sort.POPULARITY)
+    companion object {
+        private const val PAGE_SIZE = 10
+    }
+
+    private var _storyIndex = MutableLiveData<Int>()
+    val storyIndex: LiveData<Int>
+        get() = _storyIndex
+
+    private var _sortOrder = MutableLiveData(Sort.LATEST)
     val sortOrder: LiveData<Sort>
         get() = _sortOrder
 
-    private val _perfumeStoryList = MutableLiveData<List<Story?>?>()
-    val perfumeStoryList: LiveData<List<Story?>?>
+    private val _perfumeStory = MutableLiveData<Story?>()
+    val perfumeStory: LiveData<Story?>
+        get() = _perfumeStory
+
+    private val _perfumeStoryList = MutableLiveData<List<Story>>()
+    val perfumeStoryList: LiveData<List<Story>>
         get() = _perfumeStoryList
 
-    private val list = mutableListOf<Story?>()
+    private val todayAndHotStoryList = mutableListOf<Story>()
 
     fun setSortOrder(sort: Sort) {
         _sortOrder.value = sort
     }
 
-    fun getPerfumeStoryList(storyId: Int) {
+    fun getTodayAndHotStoryList(storyIdAndPerfumeIdSet: MainStorySet, storyIndex: Int) {
         viewModelScope.launch {
-            _perfumeStoryList.value = storyRepository.fetchPerfumeStoryList(storyId)
+            storyIdAndPerfumeIdSet.set?.forEach {
```

Comment:
```suggestion
            storyIdAndPerfumeIdSet.set?.forEach { (storyId, perfumeId) ->
```
it 사용은 최대한 피하는게 좋아요. storyId?.forEach { it } 이렇게 바로 확인할 수 있는 변수가 아니면 왠만하면 다 풀어서 작성해주세요